### PR TITLE
PIM-5657: Add cookbook to override job profile templates

### PR DESCRIPTION
**Pull request by SamirBoulil**

### CODE REVIEW

**Comment by a2xchip** [comment]
Cool!

File: cookbook/ui_customization/override_job_profile_templates.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
+How to override job profiles templates
+======================================
+
+It is possible to override job profile templates used to display information about the jobs (import and export jobs).
+
+It is possible to override per job templates.
+
+First, we need to create an twig template to override the default one. You can use the
```
Comment (a2xchip) [inline cookbook/ui_customization/override_job_profile_templates.rst]: an?

File: cookbook/ui_customization/override_job_profile_templates.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+The template for showing the job profile:
```
Comment (a2xchip) [inline cookbook/ui_customization/override_job_profile_templates.rst]: Sounds like "The" is surplus here

File: cookbook/ui_customization/override_job_profile_templates.rst [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        {% set removeAcl = 'pim_importexport_import_profile_remove' %}
+        {% set removeRoute = path('pim_importexport_import_profile_remove', { 'id': form.vars.value.id }) %}
+        {% set importRoute = path('pim_importexport_import_profile_index') %}
+        {% set removeMessage = 'confirmation.remove.import profile'|trans({ '%name%': form.vars.value.label }) %}
+        {% set removeSuccessMessage = 'flash.import.removed'|trans %}
+        {% endif %}
+
+        {% include 'PimImportExportBundle:JobProfile:edit.html.twig' %}
+
+In order to make override, In order to register your custom template into the PIM, you will need to create a configuration file in AcmeBundle/Resources.
```
Comment (a2xchip) [inline cookbook/ui_customization/override_job_profile_templates.rst]: ? Logical merging

Possible:

In order to override default template you will need to register your custom template by creating new configuration file in AcmeBundle\Resources

**Comment by SamirBoulil** [comment]
@a2xchip Maybe a more generic tutorial can be found here https://docs.akeneo.com/master/cookbook/ui_customization/add_a_new_tab_to_an_entity_edit_form.html#how-to-override-a-view-element-template is that what you looking for ?

File: cookbook/ui_customization/override_job_profile_templates.rst [hunk 4]
```diff
@@ -0,0 +1,64 @@
+How to override job profiles templates
+======================================
+
+It is possible to override job profile templates used to display information about the jobs (import and export jobs).
+
+First, we need to create a twig template to override the default one. You can use the
```
Comment (MarieMinasyan) [inline cookbook/ui_customization/override_job_profile_templates.rst]: The end of the sentence is missing

Comment (MarieMinasyan) [inline cookbook/ui_customization/override_job_profile_templates.rst]: Twig

File: cookbook/ui_customization/override_job_profile_templates.rst [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+       {% set removeAcl = 'pim_importexport_import_profile_remove' %}
+       {% set removeRoute = path('pim_importexport_import_profile_remove', { 'id': form.vars.value.id }) %}
+       {% set importRoute = path('pim_importexport_import_profile_index') %}
+       {% set removeMessage = 'confirmation.remove.import profile'|trans({ '%name%': form.vars.value.label }) %}
+       {% set removeSuccessMessage = 'flash.import.removed'|trans %}
+   {% endif %}
+
+   {% include 'PimImportExportBundle:JobProfile:edit.html.twig' %}
+
+In order to override default template you will need to register your custom template by creating new configuration file in ``AcmeBundle\Resources``.
```
Comment (MarieMinasyan) [inline cookbook/ui_customization/override_job_profile_templates.rst]: a default template

**Pull request closed without merging**